test/integration/puppetlabs/puppetserver/testutils.clj

```diff
@@ -39,6 +39,7 @@
    (schema/optional-key "code_id") (schema/maybe schema/Str)
    (schema/optional-key "tags") [schema/Str]
    (schema/optional-key "catalog_uuid") schema/Str
+   (schema/optional-key "catalog_format") schema/Int
```

Comment:
@KevinCorcoran I'm not very familiar with this block of tests or the current catalog work in core Puppet that has been underway, but does this seem sane to you?